### ISSUE TITLE
Readable date-time format for new created documents

### ISSUE THREAD

**Issue by jankapunkt** [openqda/openqda#24]
- Being on preparation page, creating a new file (not uploading).
- We currently get the format of an integer timestamp: `NewDocument_1715926412927_1715926413.txt`
- we should get a readable date that is also sortable: `NewDocument_20240517_082132.txt`

The pattern should be `NewDocument_<year><month><day>_<hours><minutes><seconds>.txt`

The according functions would be:

```js
/**
  * takes a date and returns a new sortable date string in the format
  * <year><month><day> where single numbers are always prefixed with a zero,
  * so each string returned will always have the same length of 8 chars.
  * @param {Date} date the date to create the string from
  * @returns {string}
  */
const getSortableDateStr = date => {
  const y = date.getFullYear();
  const m = prefixed(date.getMonth());
  const d = prefixed(date.getDate());
  return `${y}${m}${d}`;
};

/**
  * takes a date and returns a new sortable time string in the format
  * <hours><minutes><seconds> where single numbers are always prefixed with a zero,
  * so each string returned will always have the same length of 6 chars.
  * @param {Date} date the date to create the string from
  * @returns {string}
  */
const getSortableTimeStr = date => {
  const h = prefixed(date.getHours());
  const m = prefixed(date.getMinutes());
  const s = prefixed(date.getSeconds());
  return `${h}${m}${s}`;
};

/** @private */
const prefixed = num => num.toString().padStart(2, '0');
```

**Comment by alessandrobelli** [comment]
Was this address in https://github.com/openqda/openqda/pull/20 ?

**Comment by jankapunkt** [comment]
No this why I created a separate issue so we don't forget and fix this once #20 is merged
